Source/WebCore/css/CSSRayValue.cpp

```diff
@@ -34,12 +34,17 @@ namespace WebCore {
 String CSSRayValue::customCSSText() const
 {
     bool isNonDefaultSize = m_size != CSSValueClosestSide;
+    bool hasPosition = m_position;
 
     return makeString(
         "ray("_s, m_angle->cssText(),
         isNonDefaultSize ? " "_s : ""_s,
         isNonDefaultSize ? nameLiteral(m_size) : ""_s,
         m_isContaining ? " contain"_s : ""_s,
+        hasPosition ? " at "_s : ""_s,
+        hasPosition ? m_position->first().cssText() : ""_s,
+        hasPosition ? " "_s : ""_s,
+        hasPosition ? m_position->second().cssText() : ""_s,
```

Comment:
```suggestion
        hasPosition ? m_position->cssText() : ""_s,
```
I think you can just do this and get the right serialization, CSSValuePair has a customCSSText() that should work properly here.

Reply:
I tried that but it only gave the serialization for the first value for whatever reason.

Reply:
Ah I see need to set IdenticalValueSerialization.

Reply:
Fixed.